app/src/main/java/com/codesoom/assignment/DemoHttpHandler.java

```diff
@@ -0,0 +1,210 @@
+package com.codesoom.assignment;
+
+import com.codesoom.assignment.models.Task;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.*;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+enum Status {
+    OK(200), CREATED(201), NO_CONTENT(204), BAD_REQUEST(400), NOT_FOUND(404);
+
+    private final int status;
+    Status(int status) {
+        this.status = status;
+    }
+
+    int getStatus(){
+        return this.status;
+    }
+}
+
+public class DemoHttpHandler implements HttpHandler {
+    private ObjectMapper objectMapper = new ObjectMapper();
+    private List<Task> tasks = new ArrayList<>();
+    private OutputStream outputStream;
+    private Long id=1L;
```

Comment:
IDE에 포매터 플러그인을 설치하시면 이런 귀찮은 스페이스 작업을 대신해주게 됩니다.
```suggestion
    private Long id = 1L;
```